packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -68,6 +70,28 @@ describe('Sequence - ', () => {
     });
   });
 
+  it('user-defined Send', async () => {
+    const send: Send = (response, result) => {
+      response.setHeader('content-type', 'text/plain');
+      response.end(`CUSTOM FORMAT: ${result}`);
+    };
+    app.bind('send').to(send);
+
+    const client = await whenIMakeRequestTo(app);
+    await client.get('/name').expect('CUSTOM FORMAT: SequenceApp');
+  });
+
+  it('user-defined Reject', async () => {
+    const reject: Reject = (response, request, error) => {
+      response.statusCode = 418; // I'm a teapot
```

Comment:
And also because it doesn't need a message to be an error. Aside from that, do we intend to change how we present errors in response bodies as compared to loopback@3.x?